### PR TITLE
Add Release Notes for iOS Release 1.57.2

### LINKED ISSUE #8397: Release Notes for iOS Release 1.57.2
## 1.57.2

- Improved performance of closing tabs for sites where Window.open is called many times. ([#8060](https://github.com/brave/brave-ios/issues/8060))
- Fixed tab selection UI when switching between private and regular modes. ([#8146](https://github.com/brave/brave-ios/issues/8146))
- Fixed background audio not stopping after closing tab. ([#8141](https://github.com/brave/brave-ios/issues/8141))
- Fixed intermittent crash when downloading shared playlists. ([#8013](https://github.com/brave/brave-ios/issues/8013))

### PR DESCRIPTION
**Pull request by Uni-verse**
## Summary of Changes

This pull request fixes [#8397](https://github.com/brave/brave-ios/issues/8397)

## Submitter Checklist:

- [ ] *Unit Tests* are updated to cover new or changed functionality
- [ ] User-facing strings use `NSLocalizableString()`
- [ ] New or updated UI has been tested across:
  - [ ] Light & dark mode
  - [ ] Different size classes (iPhone, landscape, iPad)
  - [ ] Different dynamic type sizes

## Test Plan:

N/A

## Reviewer Checklist:

- [ ] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.57.2](https://github.com/brave/brave-ios/releases/tag/v1.57.2)
+
+- Improved performance of closing tabs for sites where Window.open is called many times. ([#8060](https://github.com/brave/brave-ios/issues/8060))
+- Fixed tab selection UI when switching between private and regular modes. ([#8146](https://github.com/brave/brave-ios/issues/8146))
+- Fixed background audio not stopping after closing tab. ([#8141](https://github.com/brave/brave-ios/issues/8141))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Maybe add `in certain cases` for the above? For example:

```
- Fixed background audio not stopping after closing tab in certain cases. ([#8141](https://github.com/brave/brave-ios/issues/8141))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.57.2](https://github.com/brave/brave-ios/releases/tag/v1.57.2)
+
+- Improved performance of closing tabs for sites where Window.open is called many times. ([#8060](https://github.com/brave/brave-ios/issues/8060))
+- Fixed tab selection UI when switching between private and regular modes. ([#8146](https://github.com/brave/brave-ios/issues/8146))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Maybe mention what the issue was? Sounds like there was issues with the wrong UI being shown and not being able to switch back to regular mode from private mode in certain cases.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.57.2](https://github.com/brave/brave-ios/releases/tag/v1.57.2)
+
+- Improved performance of closing tabs for sites where Window.open is called many times. ([#8060](https://github.com/brave/brave-ios/issues/8060))
```
Comment (kjozwiak) [inline CHANGELOG.md]: hmmm maybe rewording the above so we're not mentioning `Window.open`? Probably doesn't mean much to most users.

**Review by kjozwiak — CHANGES_REQUESTED** [review]

**Comment by Uni-verse** [comment]
@kjozwiak Thanks, changes are in. Let me know what you think.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.57.2](https://github.com/brave/brave-ios/releases/tag/v1.57.2)
+
+- Improved performance of closing all tabs on certain sites that cause many tabs to be open. ([#8060](https://github.com/brave/brave-ios/issues/8060))
```
Comment (kjozwiak) [inline CHANGELOG.md]: How about something like this?

```
- Improved performance when closing all tabs opened due to "Block Popups" being disabled on certain websites. ([#8060](https://github.com/brave/brave-ios/issues/8060))
```

**Review by kjozwiak — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (Uni-verse) [reply CHANGELOG.md]: Ok great. https://github.com/brave/brave-ios/commit/d71e90729b31dc2f49fdcd3dd345125079cb732e

**Review by kjozwiak — APPROVED** [review]
LGTM 👍

**Review by iccub — APPROVED** [review]
